ci-operator/step-registry/cucushift/installer/check/aws/private/cucushift-installer-check-aws-private-commands.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+REGION="${LEASED_RESOURCE}"
+INFRA_ID=$(jq -r '.infraID' ${SHARED_DIR}/metadata.json)
+CLUSTER_NAME="${NAMESPACE}-${UNIQUE_HASH}"
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+
+ret=0
+
+#Check there's no external load balancer created in this cluster
+CLUSTER_LB_LIST=$(mktemp)
+
+for lb in $(aws --region ${REGION} elbv2 describe-load-balancers | jq -r '.LoadBalancers[].LoadBalancerArn'); do
+    aws --region ${REGION} elbv2 describe-tags --resource-arns "$lb" | jq --arg infra "kubernetes.io/cluster/${INFRA_ID}" -ce '.TagDescriptions[].Tags[] | select( .Key == "\($infra)" and .Value == "owned")' && echo "$lb" >> "${CLUSTER_LB_LIST}"
+done
+
+for cluster_lb in $(cat $CLUSTER_LB_LIST); do 
+    LB_TYPE=$(aws --region ${REGION} elbv2 describe-load-balancers --load-balancer-arns ${cluster_lb} | jq -r '.LoadBalancers[].Scheme')
+    if [[ "${LB_TYPE}" == "internal" ]]; then
+        echo "Pass: LB ${cluster_lb} is internal."
+    else
+        echo "Error: LB ${cluster_lb} is not internal, the LB type is ${LB_TYPE}"
+        ret=$((ret + 1))
+    fi
+done
+
+
+#No public Route 53 DNS records that matches the baseDomain for the cluster
+if [[ -z ${BASE_DOMAIN} ]]; then
```

Comment:
Last time, on gcp, when installing a private cluster, specify a non-existing BASE_DOMAIN, the following check would failed, refer to [here](https://github.com/openshift/release/pull/58966/files#diff-2e7b853ef49ec709fddb2250ed207f4e8b0d8792aa2a3e84b290956027cbbd47). 

I guess we also can apply the similar fix for aws private clusters.